hudi-flink/src/test/java/org/apache/hudi/source/TestJsonStringToHoodieRecordMapFunction.java

```diff
@@ -72,7 +72,7 @@ public void testMapFunction() throws Exception {
     props.put(KeyGeneratorOptions.RECORDKEY_FIELD_OPT_KEY, "_row_key");
     props.put(KeyGeneratorOptions.PARTITIONPATH_FIELD_OPT_KEY, "current_date");
     StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
-    env.setParallelism(2);
+    env.setParallelism(1);
```

Comment:
remove the `@Disabled` as well?
Yes, this was my guess for the flaky test as well.